src/entities/InductionClass.ts

```diff
@@ -22,4 +23,10 @@ export class InductionClass {
   // Starting date of the induction cycle for said class
   @Column('date')
   endDate: string;
+
+  @Column({
+    type: 'simple-json',
```

Comment:
we should use json instead of simple-json here to rely on the native json column type

Reply:
might have to do more magic with getting types to work nicely idk